library_generation/README.md

```diff
@@ -125,15 +125,3 @@ library_generation/generate_library.sh \
 --include_samples true
 ```
 
-## An example to generate showcase client
```

Comment:
Since this part is deleted in this PR, do we have suggestions on how to generate/update showcase in this repo?